ui/js/models/resultsets_store.js

```diff
@@ -861,7 +860,7 @@ treeherder.factory('ThResultSetStore', [
 
         var getLastModifiedJobTime = function (jobList) {
             if (jobList.length > 0) {
-                return _.max(jobList.map(job => new Date(job.last_modified + 'Z')));
+                return max(jobList.map(job => new Date(job.last_modified + 'Z')));
```

Comment:
This seems like it could use `Math.max(...jobList.map(...))` too?

Reply:
Hi @edmorley , when `Math.max` applies on an array, it outputs only the maximum number. I'm not sure here, but I think we want to preserve the `Date()` objects, so that we may do other calculations on them. `lodash/max` returns the object (unmutated), while returning the one with the maximum timestamp.

```
> a = [new Date(), new Date(0)]
[ 2018-08-20T04:52:25.173Z, 1970-01-01T00:00:00.000Z ]
> a[0].getTime()
1534740745173
> Math.max(...a)
1534740745173
> Math.max(...a).getTime()
TypeError: Math.max(...).getTime is not a function
>
```

Reply:
Ah true. I guess the only other option would be to convert back to a Date again, but that would be more verbose.